tools/sigh.ts

```diff
@@ -92,6 +93,12 @@ const srcExclude = /\b(node_modules|deps|build|gen|dist|third_party|javaharness|
 // RE pattern to exclude when finding within project built files.
 const buildExclude = /\b(node_modules|deps|src|third_party|javaharness|Kotlin)\b/;
 
+// Standard flags for invoking node.
+const nodeFlags = [
+  '--experimental-modules',
+  '--loader', fixPathForWindows(path.join(__dirname, '../tools/custom-loader.mjs'))
```

Comment:
If we used `path.join(__dirname, 'tools', 'custom-loader.mjs')` would this need the fixPathForWindows?

I think node's path lib is platform independent.

Reply:
I don't actually know, and without easy access to a Windows dev machine it's hard to make changes for Windows-specific things :-/